sdk/storage/azure-storage-blob-changefeed/azure/storage/blob/changefeed/_change_feed_client.py

```diff
@@ -0,0 +1,120 @@
+
+    :param str account_url:
+        The URI to the storage account.
+    :param credential:
+        The credentials with which to authenticate. This is optional if the
+        account URL already has a SAS token. The value can be a SAS token string, an account
+        shared access key, or an instance of a TokenCredentials class from azure.identity.
+        If the URL already has a SAS token, specifying an explicit credential will take priority.
+    :keyword str secondary_hostname:
+        The hostname of the secondary endpoint.
+
+    .. admonition:: Example:
+
+        .. literalinclude:: ../samples/change_feed_samples.py
+            :start-after: [START create_change_feed_client]
+            :end-before: [END create_change_feed_client]
+            :language: python
+            :dedent: 8
+            :caption: Creating the ChangeFeedClient from a URL to a public blob (no auth needed).
+    """
+    def __init__(
+            self, account_url,  # type: str
+            credential=None,  # type: Optional[Any]
+            **kwargs  # type: Any
+        ):
+        # type: (...) -> None
+        self._blob_service_client = BlobServiceClient(account_url, credential, **kwargs)
+
+    @classmethod
+    def from_connection_string(
+            cls, conn_str,  # type: str
+            credential=None,  # type: Optional[Any]
+            **kwargs  # type: Any
+        ):  # type: (...) -> BlobServiceClient
+        """Create ChangeFeedClient from a Connection String.
+
+        :param str conn_str:
+            A connection string to an Azure Storage account.
+        :param credential:
+            The credentials with which to authenticate. This is optional if the
+            account URL already has a SAS token, or the connection string already has shared
+            access key values. The value can be a SAS token string, an account shared access
+            key, or an instance of a TokenCredentials class from azure.identity.
+            Credentials provided here will take precedence over those in the connection string.
+        :returns: A change feed client.
+        :rtype: ~azure.storage.blob.changefeed.ChangeFeedClient
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/blob_samples_authentication.py
+                :start-after: [START auth_from_connection_string]
+                :end-before: [END auth_from_connection_string]
+                :language: python
+                :dedent: 8
+                :caption: Creating the BlobServiceClient using account_key as credential.
+        """
+        account_url, secondary, credential = parse_connection_str(conn_str, credential, 'blob')
+        if 'secondary_hostname' not in kwargs:
+            kwargs['secondary_hostname'] = secondary
+        return cls(account_url, credential=credential, **kwargs)
+
+    def list_changes(self, **kwargs):
+        # type: (Optional[datetime], Optional[datetime], **Any) -> ItemPaged[BlobProperties]
```

Comment:
ItemPaged[BlobProperties] is this correct?

Reply:
good catch, it should be dict